src/components/dev-dot-content/mdx-components/index.tsx

```diff
@@ -0,0 +1,2 @@
+export * from './mdx-lists'
+export * from './mdx-tabs'
```

Comment:
I've added a barrel file for MDX components shared between docs and tutorials. Expecting this file to grow as we work on 🎟️ [reconcile MDX components for /docs and /tutorials](https://app.asana.com/0/1201987349274776/1202052090505082/f) (post June 20).